python-backend/app/api/mines/mine/models/mine_identity.py

```diff
@@ -62,24 +61,41 @@ def json(self):
         return {
             'guid':
             str(self.mine_guid),
+            'mine_name':
+            self.mine_name,
+            'mine_no':
+            self.mine_no,
+            'mine_note':
+            self.mine_note,
+            'major_mine_ind':
+            self.major_mine_ind,
+            'region_code':
+            self.mine_region,
```

Comment:
It doesn't look like these short lines need to be wrapped. The lines below this comment are longer and aren't wrapped.

Reply:
Ive tried to manually make them stay on one line but the formatter forces them to another line.